open_schools_platform/organization_management/circles/tests/utils.py

```diff
@@ -1,25 +1,36 @@
+from typing import Any
+
 from django.contrib.gis.geos import Point
 
 from open_schools_platform.organization_management.circles.models import Circle
 from open_schools_platform.organization_management.circles.services import create_circle, add_student_to_circle
 from open_schools_platform.organization_management.employees.tests.utils import create_test_employee
+from open_schools_platform.organization_management.organizations.models import Organization
 from open_schools_platform.organization_management.organizations.tests.utils import create_test_organization
 from open_schools_platform.user_management.users.models import User
 from open_schools_platform.student_management.students.services import create_student
 
 
-def create_test_circle() -> Circle:
-    organization = create_test_organization()
-    circle = create_circle(organization=organization, name="test_circle", address="d", capacity=0, description="alalal",
-                           location=Point(0.0, 0.0))
+def create_test_circle(organization: Organization = None, address: str = "address",
+                       location: Any = Point(0.0, 0.0)) -> Circle:
+    if not organization:
+        organization = create_test_organization()
+    circle = create_circle(
+        organization=organization,  # type: ignore
```

Comment:
why type ignore?

Reply:
Because mypy does not want organization argument to be optional (It is optional now, because I provided default None value for it)